geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/ExportStackTraceCommand.java

```diff
@@ -50,6 +53,10 @@
   public static final String STACK_TRACE_FOR_MEMBER = "*** Stack-trace for member ";
   private final GetStackTracesFunction getStackTracesFunction = new GetStackTracesFunction();
 
+  DateTimeFormatter formatter =
```

Comment:
Small Nitpick: now that you're using `DateTimeFormatter` (immutable and thread safe) instead of `SimpleDateFormat`, you might want to create the formatter instance just once and re-use it instead of creating it every time the command is executed.